### PR TITLE
[VUMM-653] Modify IntegrationTest.java to run in each PR

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

**Review by evgeniilezhnin — APPROVED** [review]

**Comment by anandjakhaniya** [comment]
@jkwatson-verta .

**Comment by anandjakhaniya** [comment]
!rebuild

File: backend/server/src/test/java/ai/verta/modeldb/IntegrationTest.java [hunk 1]
```diff
@@ -180,22 +195,22 @@ private CreateExperimentRun createExperimentRunRequest(String projectId, String
...
+    /*observations.add(
+    Observation.newBuilder()
+        .setArtifact(
+            Artifact.newBuilder()
+                .setKey("Google developer Observation artifact")
+                .setPath("This is data artifact type in Google developer Observation artifact")
+                .setArtifactType(ArtifactType.DATA)
+                .build())
+        .setTimestamp(Calendar.getInstance().getTimeInMillis())
+        .build());*/
```
Comment (jkwatson-verta) [inline backend/server/src/test/java/ai/verta/modeldb/IntegrationTest.java]: should we just delete this code rather than comment it out?

Comment (anandjakhaniya) [reply backend/server/src/test/java/ai/verta/modeldb/IntegrationTest.java]: Removed

**Pull request merged**